### ISSUE TITLE
feature request: Fiat amount on coins list

### ISSUE THREAD

**Issue by emergent-reasons** [Electron-Cash/Electron-Cash#1710]
Use case: I often want to send a single coin if possible. With fusion and shuffle, this is often possible. However coins list does not have fiat amount so it's a 2 or 3 step process to figure out which one is closest.

**Comment by cculianu** [comment]
Yeah coins list was the readheaded stepchild of the tabs until it took front and center last year with cashshuffle.

This is a good feature request.  Will add it to The List of Things Yet To Be Compleat.
